GiEnJul/ClientApp/src/routes/registerGiver/SummaryRegistration.tsx

```diff
@@ -3,25 +3,38 @@ import * as React from 'react';
 import { Button, Grid, TextField, FormControl} from '@material-ui/core';
 import useStyles from './Styles';
 
-
 type Props = {
     nextStep: () => void,
     prevStep: () => void,
-    submit: { location?: string; fullname?: string; email?: string; phoneNumber?: number; maxRecievers?: number };
-    values: { location?: string; fullname?: string; email?: string; phoneNumber?: number; maxRecievers?: number; familyType?: string}
+    submit: { location?: string; fullname?: string; email?: string; phoneNumber?: number; maxRecievers?: number; confirmationOK: boolean},
+    values: { location?: string; fullname?: string; email?: string; phoneNumber?: number; maxRecievers?: number; familyType?: string, confirmationOK: boolean},
+    callingback: (e: boolean) => void
+    
 }
 
-const SummaryRegistration: React.FC<Props> = ({ nextStep, prevStep, submit, values }) => {
+const SummaryRegistration: React.FC<Props> = ({ nextStep, prevStep, submit, values, callingback }) => {
+
+    const trigger = (b: boolean) => {
+        callingback(b);
+    }
 
-    const Submit = (e: any) => {
+    const Submit = async( e: any) => {
         e.preventDefault();
-        fetch('https://localhost:5001/api/giver', {
+        await fetch('https://localhost:5001/api/giver', {
```

Comment:
Can change this to to keep the link relative
```suggestion
        await fetch('/api/giver', {
```

Reply:
That would be better yes, will fix